docs/src/main/sphinx/release/release-437.md

```diff
@@ -0,0 +1,12 @@
+# Release 437 (17 Jan 2024)
+
+## General
+
+* Add support for `char(n)` values in {func}`to_utf8`. ({issue}`20158`)
+* Add support for `char(n)` values in {func}`lpad`. ({issue}`16907`)
+* Fix [table function-based](/functions/table) materialized views always being
```

Comment:
I think this assumes too much .. maybe

Prevent permanent staleness of materialized views that use query pass-through table functions.


Wdyt about this suggestion instead @homar 



Also

Reply:
@colebow ?